test/README.md

```diff
@@ -200,11 +200,10 @@ Change directory to the test directory:
 cd test
 ```
 
-To setup up minikube for drenv run once before starting any
-environment:
+To setup up specified kubernetes cluster for drenv, run once before starting any environment:
```

Comment:
drenv setup does not set up a kubernetes cluster, but sets up the *host* for running the specific drenv environment file. The environment file  is describing one or more kubernetes clusters.

Reply:
Updated.